src/vorta/utils.py

```diff
@@ -20,9 +20,9 @@
 from PyQt5.QtGui import QIcon
 from PyQt5 import QtCore
 import subprocess
-import keyring
-from vorta.keyring_db import VortaDBKeyring
-
+import vorta.keyring.keyring_core as keyring
+from vorta.keyring.keyring_db import VortaDBKeyring
+import secretstorage
```

Comment:
if it is imported here, even darwin will require it - is this intended?

Reply:
Good point, I will change that.